src/cascadia/TerminalSettingsModel/Profile.cpp

```diff
@@ -343,6 +183,7 @@ void Profile::LayerJson(const Json::Value& json)
 
     // Profile-specific Settings
     JsonUtils::GetValueForKey(json, NameKey, _Name);
+    JsonUtils::GetValueForKey(json, UpdatesKey, _Updates);

```

Comment:
_Interesting!_ Curious: What happens if a user happens to use `updates` in their settings.json?

Reply:
Nothing should happen. The `Updates()` getter is only called for fragments inside `SettingsLoader::MergeFragmentsIntoUserProfiles()`.